lib/request-validation.ts

```
@@ -1,38 +1,70 @@
+      ) {
         errors.push(`Disallowed header: ${header}`);
       }
     }
   }
 
   if (config.maxBodySize && request.body) {
-    const body = await request.text();
-    if (body.length > config.maxBodySize) {
-      errors.push(`Request body exceeds max size of ${config.maxBodySize} bytes`);
+    const contentLength = request.headers.get("content-length");
+    if (contentLength && parseInt(contentLength) > config.maxBodySize) {
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)